doc/deprecated.md

```diff
@@ -50,6 +50,21 @@ namespace. For status on this see [#8234].
 
 </details>
 
+<details>
+<summary>Pubsub: remove legacy admin APIs</summary>
+<br>
+
+We will remove the hand-written versions of the Pub/Sub admin APIs. These have
+been superseded by versions generated automatically from the service
+definitions. The new APIs can be found in the
+[`google/cloud/pubsub/admin`](https://github.com/googleapis/google-cloud-cpp/tree/main/google/cloud/pubsub/admin)
+tree and within the `google::cloud::pubsub_admin` namespace. Starting with the
+v1.32.0 release, and depending on your compiler settings, using the old
+classes/functions may elicit a deprecation warning. See [#7356] for more
```

Comment:
Did we lose a link here?  And, it looks like it was a link to a Spanner issue.

Reply:
It should still be linked below.

Reply:
agh I see it's spanner in the original too.

Reply:
Ok added the new issue